Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1765,6 +1790,22 @@ eval BackPressed state = do
     RideRating ->     do
                       _ <- pure $ updateLocalStage RideCompleted
                       continue state {props {currentStage = RideCompleted}}
+    EditingDestinationLoc -> do
+      void $ pure $ performHapticFeedback unit
+      exit EditDestBackPressed
+    ConfirmEditDestinationLoc -> do
+      void $ pure $ performHapticFeedback unit
+      exit EditDestBackPressed
+    ConfirmingEditDestinationLoc -> do
+      void $ pure $ performHapticFeedback unit
+      exit EditDestBackPressed
+    RevisedEstimate -> do
+      void $ pure $ performHapticFeedback unit
+      exit EditDestBackPressed
+    FavouriteLocationModelEditDest -> do
+                      void $ pure $ performHapticFeedback unit
+                      _ <- pure $ updateLocalStage (if state.props.isSearchLocation == NoView then HomeScreen else EditingDestinationLoc)
```

Comment:
can we store `updateLocalStage (if state.props.isSearchLocation == NoView then HomeScreen else EditingDestinationLoc)` in a variable and use it in continue state also